docs/reference/modules/module.md

```diff
@@ -0,0 +1,103 @@
+# `@apostrophecms/module`
+
+This module is the foundation of all other Apostrophe modules. It is not *directly* used for any functionality, but [every other module inherits its behavior and functionality](/guide/modules.md#module-inheritance). Every other module has access to use the features documented here. Critically, this module includes initialization behavior that all modules need to start up correctly and register features.
+
+:::warning
+**This module should almost never be configured directly in project code.** In other words, there should almost never be a `modules/@apostrophecms/module/index.js` file in a project codebase or `@apostrophecms/module: {}` line in an `app.js` file. The only reason to do so would be to add a feature that every other module will need.
+
+**Instead, configure the options below on other modules as needed** (e.g., configure the `alias` option on a module to create an easy reference to *that* module). Similarly, run the methods below as needed from the most appropriate module (e.g., use `self.email` from a contact page-type module).
+:::
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+|`alias` | String | A name to use for quick access to the module on the `apos` object (e.g., `'book'` to access a module on `self.apos.book`). Otherwise the longer syntax is necessary (e.g., `self.apos.modules.book`). |
+|`templateData` | Object | An object of properties to include on the `data` object in templates belonging to that module. |
+
+## Featured methods
+
+The following methods belong to this module and may be useful in project-level code. As noted above, *they should be used via another, more appropriate module,* rather than directly called from this module. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/module/index.js) for all methods that belong to this module.
```

Comment:
They cannot be directly called on this module, since it is never instantiated, it is just an abstract base class.